app/modules/import-export-customization/assets/js/shared/hooks/use-context-detection.js

```
@@ -0,0 +1,18 @@
+import { useContext } from 'react';
+import { ImportContext } from '../../import/context/import-context';
+import { ExportContext } from '../../export/context/export-context';
+
+export default function useContextDetection() {
+	const importContext = useContext( ImportContext );
+	const exportContext = useContext( ExportContext );
+
+	if ( importContext ) {
+		return { isImport: true, contextData: importContext };
+	}
+
+	if ( exportContext ) {
+		return { isImport: false, contextData: exportContext };
+	}
+
+	return { isImport: null, contextData: null };
```

let's throw an error, as we should never reach this code